<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionUpdate"/>
    <addaction name="actionSaveWindow"/>
    <addaction name="actionMinimize"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAboutApp"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuPomoc"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUpdate">
   <property name="text">
    <string>Aktualizuj</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Wyj≈õcie</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>O Qt</string>
   </property>
  </action>
  <action name="actionAboutApp">
   <property name="text">
    <string>Pomoc</string>
   </property>
  </action>
  <action name="actionSaveWindow">
   <property name="text">
    <string>Zapisz zrzut ekranu</string>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="text">
    <string>Minimalizuj</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
